src/components/shared/SampleTooltipContents.js

```diff
@@ -33,6 +35,8 @@ type RestProps = {|
   +categories: CategoryList,
   +rangeFilteredThread: Thread,
   +implementationFilter: ImplementationFilter,
+  +zeroAt: Milliseconds,
+  +interval: Milliseconds,
```

Comment:
Unless I'm missing something, these new properties should rather be in Props below, it's not used by the `SampleTooltipRestContents` compoenent.

Reply:
Ah yeah, moved it to the right type.